ietf/templates/doc/opengraph.html

```diff
@@ -0,0 +1,16 @@
+{# Copyright The IETF Trust 2016-2020, All Rights Reserved #}
+{% load origin %}
+{% load static %}
+{% origin %}
+<meta property="og:title" content="{{ doc.title }} ({% if doc.get_state_slug == 'rfc' and not snapshot %}RFC {{ rfc_number }}{% else %}{{ name }}-{{ doc.rev }}{% endif %})">
```

Comment:
Including draft name here is going to be very noisy.

```suggestion
<meta property="og:title" content="({% if doc.get_state_slug == 'rfc' and not snapshot %}RFC {{ rfc_number }}: {% endif %}){{ doc.title }}">
```